pages/resources/[page].js

```diff
@@ -62,6 +69,26 @@ function Resources() {
     paid: paid || '',
   };
 
+  const handleLogin = value => loginUser(value);
+
+  const handleLoginSuccess = ({ token }) => {
+    setAuthCookies({ token });
+  };
+
+  const handleVote = (/* voteDirection, updateVoteCountFunc */) => {
+    setErrorMessage(null);
+    if (!hasValidAuthToken()) {
+      setIsModalOpen(true);
+    }
+    // Make api calls to update vote
+
+    // If successful, set new vote count
```

Comment:
FWIW. - Documentation for the resources API is here: https://resources.operationcode.org/

I guess that's outside the scope of the issue though... Lemme know if you want me to open a new issue or if you want to resolve it here and I'll count this PR as resolving 2 issues.

Reply:
Yeah. I actually worked on the endpoint for upvote/downvote so I am familiar with it. Would it be okay if I did that in a separate PR so it would count towards official hacktoberfest progress?

Reply:
Of course!

Reply:
Thx!🙏